<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pb_selecionar_arquivo_entrada">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Arquivo Entrada</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbx_simulated_annealing">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>571</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Simulated Annealing</string>
    </property>
    <widget class="QLabel" name="lbl_num_iteracoes">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Numero de Iteracoes:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ln_num_iteracoes">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>10000</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ln_taxa_resfriamento">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>0.9</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_taxa_resfriamento">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Taxa resfriamento:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_temperatura_inicial">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Temperatura Inicial:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ln_temperatura_inicial">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>120</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_executar">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Executar</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtEdit_resposta">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>551</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="ln_saida_parser">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>421</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>592</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
